server.js

```diff
@@ -6663,6 +6663,48 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// JetBrains Plugins repository version integration
+camp.route(/^\/jetbrains\/plugin\/(v)\/([^/]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var pluginId = match[2];
+  var type = match[1];
+  var format = match[3];
```

Comment:
Since you're not doing anything with `type`, you can remove it as a capture group in the regular expression. Just remove the parentheses around it. Then change `match[2]` to `match[1]` and change `match[3]` to `match[2]`.

Reply:
See https://github.com/badges/shields/pull/1164#discussion_r144404398